lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -331,15 +331,14 @@ def install_puppet_on(hosts, opts={})
         # @option opts [String] :puppet_gem_version Version of puppet to install via gem if no puppet-agent package is available
         # @option opts [String] :mac_download_url Url to download msi pattern of %url%/puppet-agent-%version%.msi
         # @option opts [String] :win_download_url Url to download dmg pattern of %url%/puppet-agent-%version%.msi
-        # @option opts [String] :puppet_collection Defaults to 'pc1'
+        # @option opts [String] :puppet_collection Defaults to nil
         # @option opts [Boolean] :run_in_parallel Whether to run on each host in parallel.
         #
         # @return nil
         # @raise [StandardError] When encountering an unsupported platform by default, or if gem cannot be found when default_action => 'gem_install'
         # @raise [FailTest] When error occurs during the actual installation process
         def install_puppet_agent_on(hosts, opts = {})
           opts = FOSS_DEFAULT_DOWNLOAD_URLS.merge(opts)
-          opts[:puppet_collection] ||= 'pc1' #hi!  i'm case sensitive!  be careful!
```

Comment:
So we're just not setting a default value here, but folks can still pass this value in. If it's set to `pc1` (case sensitive), it'll do the same as previous, which is enable the pc1 repos. If left blank, it'll default to the puppet rolling repos. Other than pc1, possible values are going to correspond to any major version series. Right now the only one out is `5`, but once puppet6 is live, it could be set to `6`, etc.